sandbox/anvi-script-reformat-fasta

```diff
@@ -113,12 +116,21 @@ def reformat_FASTA(args):
             total_num_contigs_removed += 1
             continue
 
-        percentage_of_gaps = fasta.seq.count('-') * 100.0 / l
+        num_gaps = fasta.seq.count('-')
+        sequence_num_gaps_list = [fasta.id, num_gaps, fasta.seq]
+        num_gaps_list.append(sequence_num_gaps_list)
+        percentage_of_gaps = num_gaps * 100.0 / l
```

Comment:
`sequence_num_gaps_list` does not look like a necessary variable. This change would remove the need for it completely:

``` python
num_gaps_list.append([fasta.id, num_gaps, fasta.seq], )
```

That list variable looks more like `gaps_info_list` than `num_gaps_list` to me.

Another note, that list is populated even if `args.export_gap_counts_table` is False, but it is only used for anything if `args.export_gap_counts_table` is True down below.

Reply:
Thanks Meren, I just fixed these issues in the most recent commit.